### PR TITLE
Permission Matrix updates - Credit Account

### PR DESCRIPTION
**Pull request by krishnan-aot**
# Description

- Proper configuration of individual credit account components
- View LOA update on Finance user

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

## Further comments

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://onroutebc-1584-frontend.apps.silver.devops.gov.bc.ca)
- [Vehicles](https://onroutebc-1584-vehicles.apps.silver.devops.gov.bc.ca/api)
- [Dops](https://onroutebc-1584-dops.apps.silver.devops.gov.bc.ca/api)
- [Policy](https://onroutebc-1584-policy.apps.silver.devops.gov.bc.ca/api)
- [Scheduler](https://onroutebc-1584-scheduler.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/onroutebc/actions/workflows/analysis.yml)

After merge, new images are promoted to:
- [Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge.yml)

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://onroutebc-1584-frontend.apps.silver.devops.gov.bc.ca)
- [Vehicles](https://onroutebc-1584-vehicles.apps.silver.devops.gov.bc.ca/api)
- [Dops](https://onroutebc-1584-dops.apps.silver.devops.gov.bc.ca/api)
- [Policy](https://onroutebc-1584-policy.apps.silver.devops.gov.bc.ca/api)
- [Scheduler](https://onroutebc-1584-scheduler.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/onroutebc/actions/workflows/analysis.yml)

After merge, new images are promoted to:
- [Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge.yml)

### CODE REVIEW

File: frontend/src/features/settings/components/dashboard/ManageSettingsDashboard.tsx [hunk 1]
```diff
@@ -41,9 +54,10 @@ export const ManageSettingsDashboard = React.memo(() => {
   const showCreditAccountTab = usePermissionMatrix({
     featureFlag: "CREDIT-ACCOUNT",
     permissionMatrixFeatureKey: "MANAGE_SETTINGS",
-    permissionMatrixFunctionKey: "VIEW_CREDIT_ACCOUNT_TAB",
+    permissionMatrixFunctionKey: getPermissionMatrixFunctionKey(),
     additionalConditionToCheck: () =>
-      // Show the tab for all users if the user is
+      // Show the tab if there is a credit account or if the user is a finance user.
+      // Todo: Display info box if there is no credit account.
```
Comment (zgong-gov) [inline frontend/src/features/settings/components/dashboard/ManageSettingsDashboard.tsx]: Has this "todo" been completed?

File: frontend/src/features/settings/pages/CreditAccountMetadataComponent.tsx [hunk 2]
```diff
@@ -20,11 +20,13 @@ export const CreditAccountMetadataComponent = ({
         />
       );
     } else {
+      // Todo: Display info box for non-finance staff users who
```
Comment (zgong-gov) [inline frontend/src/features/settings/pages/CreditAccountMetadataComponent.tsx]: Has this "todo" been completed?

File: frontend/src/features/settings/components/dashboard/ManageSettingsDashboard.tsx (hunk 1 above)
Comment (krishnan-aot) [reply frontend/src/features/settings/components/dashboard/ManageSettingsDashboard.tsx]: No, I've created a ticket for it. [ORV2-2771](https://jira.th.gov.bc.ca/browse/ORV2-2771)

File: frontend/src/features/settings/pages/CreditAccountMetadataComponent.tsx (hunk 2 above)
Comment (krishnan-aot) [reply frontend/src/features/settings/pages/CreditAccountMetadataComponent.tsx]: No, I've created a ticket for it. [ORV2-2771](https://jira.th.gov.bc.ca/browse/ORV2-2771)

File: frontend/src/features/settings/components/dashboard/ManageSettingsDashboard.tsx (hunk 1 above)
Comment (zgong-gov) [reply frontend/src/features/settings/components/dashboard/ManageSettingsDashboard.tsx]: ok

File: frontend/src/features/settings/pages/CreditAccountMetadataComponent.tsx (hunk 2 above)
Comment (zgong-gov) [reply frontend/src/features/settings/pages/CreditAccountMetadataComponent.tsx]: ok

**Review by zgong-gov — APPROVED** [review]

**Pull request merged**